### PR TITLE
pep8 fix and Vagrant update

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

**Comment by benethor** [comment]
:+1:

File: alamo_scheduler/scheduler/ap_scheduler.py [hunk 1]
```diff
@@ -7,14 +8,19 @@
...
 
 from apscheduler.schedulers.asyncio import AsyncIOScheduler
+
 from alamo_scheduler.fixtures.checks import load_data_to_memory, SAMPLE_QUERY
 
+
 def tick(check, test):
     if 'check_{}'.format(test) == check['name']:
-        print('{} Tick! Interval is: {}. The time is: {}'.format(check['name'], check['interval'] ,datetime.now()))
+        print('{} Tick! Interval is: {}. The time is: {}'.
```
Comment (deejay1) [inline alamo_scheduler/scheduler/ap_scheduler.py]: logging ;)

Comment (ch40s3k) [inline alamo_scheduler/scheduler/ap_scheduler.py]: That are fixes for pep8 ;)

**Comment by deejay1** [comment]
:+1: But after merging the fixtures branch do dev fix it (and squash for some better code in history :P)

**Pull request merged**